### PR TITLE
Ardenian

### PR DESCRIPTION
**Pull request by Ardenian**
Pull Request for Ticket #157

### CODE REVIEW

File: src/mods/chat.js [hunk 1]
```diff
@@ -744,29 +744,29 @@ idrinth.chat = {
                 'chat-service/create/',
                 idrinth.chat.joinCallback,
                 function ( reply ) {
-                    idrinth.core.alert ( idrinth.text.get ( "chat.message.createFail" ) );
+                    idrinth.core.alert ( idrinth.text.get ( "chat.error.create" ) );
                 },
                 function ( reply ) {
-                    idrinth.core.alert ( idrinth.text.get ( "chat.message.createFail" ) );
+                    idrinth.core.alert ( idrinth.text.get ( "chat.error.createFail" ) );
```
Comment (Idrinth) [inline src/mods/chat.js]: createFail should only be create

File: src/mods/chat.js [hunk 2]
```diff
@@ -744,29 +744,29 @@ idrinth.chat = {
...
     joinCallback: function ( reply ) {
         if ( !reply ) {
-            idrinth.core.alert ( idrinth.text.get ( "chat.message.joinFailMoment" ) );
+            idrinth.core.alert ( idrinth.text.get ( "chat.error.joinMoment" ) );
             return;
         }
         reply = JSON.parse ( reply );
         if ( !reply ) {
-            idrinth.core.alert ( idrinth.text.get ( "chat.message.joinFailMoment" ) );
+            idrinth.core.alert ( idrinth.text.get ( "chat.error.joinMoment" ) );
```
Comment (Idrinth) [inline src/mods/chat.js]: could be merged with chat.error.join, the difference seems tiny

File: src/mods/chat.js [hunk 3]
```diff
@@ -849,7 +849,7 @@ idrinth.chat = {
             idrinth.chat.join ( data.data );
             return;
         }
-        idrinth.core.alert ( idrinth.text.get ( "chat.message.loginFailMoment" ) );
+        idrinth.core.alert ( idrinth.text.get ( "chat.error.loginMoment" ) );
```
Comment (Idrinth) [inline src/mods/chat.js]: Moment is likely unneeded, please have a look if it can be combined with chat.error.login

**Review by Idrinth — CHANGES_REQUESTED** [review]

File: src/mods/text.js [hunk 4]
```diff
@@ -130,7 +132,12 @@ idrinth.text = {
                 enableFavRequest: 'Enable Auto-Raid-Request for Favorites?',
                 favIdToJoin: 'FavoriteIds to join (separate multiple by comma)',
                 worldserver: 'Worldserver?',
-                disableAutoJoinSpecific: 'Disable Autojoining for specific raids'
+                disableAutoJoinSpecific: 'Disable Autojoining for specific raids',
+                chat:{
```
Comment (Idrinth) [inline src/mods/text.js]: try notification here instead of duplicating it below. You might also want to move the whole group into data.chat

**Review by Idrinth — CHANGES_REQUESTED** [review]

File: src/mods/chat.js (hunk 1 above)
Comment (Ardenian) [reply src/mods/chat.js]: Fixed, thank you

File: src/mods/chat.js (hunk 2 above)
Comment (Ardenian) [reply src/mods/chat.js]: Replaced, thank you

File: src/mods/chat.js (hunk 3 above)
Comment (Ardenian) [reply src/mods/chat.js]: Replaced, thank you. Didn't even show up in my text collection, think it got under the wheels somewhere, going to check the difference

File: src/mods/text.js (hunk 4 above)
Comment (Ardenian) [reply src/mods/text.js]: Done, thank you

**Comment by Idrinth** [comment]
LGTM
Seems to work fine

**Review by Idrinth — APPROVED** [review]

**Pull request merged**